packages/cli/src/commands/deployment/deploy.ts

```diff
@@ -33,6 +33,18 @@ export default class Deploy extends Command {
     queryVersion: Flags.string({description: 'Enter query-version', required: false}),
     dict: Flags.string({description: 'Enter dictionary', required: false}),
     endpoint: Flags.string({description: 'Enter endpoint', required: false}),
+    //indexer set up flags
+    unsafe: Flags.boolean({description: 'Enable indexer unsafe', required: false}),
+    batchSize: Flags.integer({description: 'Enter batchSize from 1 to 30', required: false}),
+    indexerSubscription: Flags.boolean({description: 'Enable Indexer subscription', required: false}),
+    historicalData: Flags.boolean({description: 'Enable Historical Data', required: false}),
+    workers: Flags.integer({description: 'Enter worker threads from 1 to 30', required: false}),
+    //query flags
+    queryUnsafe: Flags.boolean({description: 'Enable indexer unsafe', required: false}),
+    querySubscription: Flags.boolean({description: 'Enable Query subscription', required: false}),
+    queryTimeout: Flags.integer({description: 'Enter timeout from 1000ms to 60000ms', required: false}),
+    maxConnection: Flags.integer({description: 'Enter MaxConnection from 1 to 10', required: false}),
+    Aggregate: Flags.boolean({description: 'Enable Aggregate', required: false}),
```

Comment:
I think we should prefix all of these with `indexer` or `query`. We don't know what flags we will add in the future that could conflict